### PR TITLE
performance bench: add bench cache (On correct branch)

### PR DESCRIPTION
**Pull request by H01001000**
Notice on benchies branch

This PR is the foundation of https://github.com/discordeno/discordeno/pull/2546, which add a script that will cache a guild's object and run on a schedule (per day for now)

But need to discuss and modify the actual guild and channel to fetch

This also removes all other files,

### CODE REVIEW

File: cache/checkAndUpdateObjects.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    }
+    if (typeof object[key] === "object" && object[key] !== null) {
+      sortValues(object[key])
+    }
+  }
+}
+
+const guild = (await bot.rest.runMethod(bot.rest, "GET", bot.constants.routes.GUILD(guildId, true)))
+sortValues(guild)
+await Deno.writeTextFile(`cache/cachedObject/guild.json`, JSON.stringify(guild, undefined, 2))
```
Comment (itohatweb) [inline cache/checkAndUpdateObjects.ts:32]: This violates Discord ToS.

File: cache/checkAndUpdateObjects.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+const channels = (await bot.rest.runMethod(bot.rest, "GET", bot.constants.routes.GUILD_CHANNELS(guildId))) as any[];
+
+const channelNames = ["rules", "announcement-channel", "moderator-channel", "text-channel", "stage-channel", "voice-channel"]
+
+for (const channelName of channelNames) {
+  const channel = channels.find((channel) => channel.name === channelName)
+  sortValues(channel)
+  await Deno.writeTextFile(`cache/cachedObject/${channelName}.json`, JSON.stringify(channel, undefined, 2))
```
Comment (itohatweb) [inline cache/checkAndUpdateObjects.ts:40]: Also violates ToS

File: cache/checkAndUpdateObjects.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+const user = await bot.rest.runMethod(bot.rest, "GET", bot.constants.routes.USER(bot.id));
+sortValues(user)
+await Deno.writeTextFile(`cache/cachedObject/user.json`, JSON.stringify(user, undefined, 2))
```
Comment (itohatweb) [inline cache/checkAndUpdateObjects.ts:45]: This violates ToS too

File: LICENSE [hunk 4]
```diff
@@ -1,201 +0,0 @@
-                                 Apache License
```
Comment (itohatweb) [inline LICENSE:1]: Why was the license removed?

**Review by itohatweb — CHANGES_REQUESTED** [review]
I'm not sure about this PR since it violates Discords ToS.

File: cache/checkAndUpdateObjects.ts (hunk 1 above)
Comment (H01001000) [reply cache/checkAndUpdateObjects.ts:52]: added censor function from benchmark repo

**Pull request merged**